src/server/views/api/queueRetry.js

```
@@ -0,0 +1,35 @@
+/**
+ * Retry all queue failed jobs
+ * @param {*} req 
+ * @param {*} res 
+ */
+async function handler(req, res) {
+  const { queueName, queueHost } = req.params;
+
+  const { Queues } = req.app.locals;
+
+  const queue = await Queues.get(queueName, queueHost);
+  if (!queue) return res.status(404).send({error: 'queue not found'});
+
+  const promises = [];
+  let jobs;
+  if (queue.IS_BEE) {
+    jobs = await queue.getJobs('failed');
```

This will only return up to 100 jobs by default, so it's a bit misleading to suggest that this will retry all jobs. Bee-queue does not provide functionality to let you query all jobs as that can result in an unbounded (realistically it's memory-bounded) time period where the redis server is unresponsive. Additionally, having this page through all pages could result in an (actually) unbounded operation that runs in perpetuity (e.g. if there's another actor adding new jobs or failing jobs and there's no limit).

I'd like to suggest that this isn't a feature we should be supporting in bee-queue. Since bull-queue already made its bed by supporting a `getFailed` operation, I don't mind adding this functionality there.